example/notify_inval_entry.c

```diff
@@ -258,8 +268,22 @@ static void* update_fs_loop(void *data) {
         update_fs();
         if (!options.no_notify && lookup_cnt) {
             if(options.only_expire) {
-                assert(fuse_lowlevel_notify_expire_entry
-                   (se, FUSE_ROOT_ID, old_name, strlen(old_name), FUSE_LL_EXPIRE_ONLY) == 0);
+                int ret = fuse_lowlevel_notify_expire_entry
+                   (se, FUSE_ROOT_ID, old_name, strlen(old_name));
+               // forget is not being called
+               // the dentry is running just into a timeout
+	       if (ret == -ENOSYS) {
+		 printf("fuse_lowlevel_notify_expire_entry not supported by kernel\n");
+		 printf("Stopping mountpoint...\n");
+		 raise(SIGINT);
+		 return NULL;
```

Comment:
This leaks old_name - break the look. Is the raise needed? Call fuse_session_exit()?

Is raise() needed? Set se->error() before fuse_session_exit()?